### DISCUSSION TITLE
Appium 2.0 release date

### DISCUSSION THREAD

**Discussion by omprakashchavan01** [appium/appium#15828] [Release]
Is 15th September final?

Pardon if I'm using wrong forum to ask such question.

**Comment by jlipps** [comment]
No, it's not final, and it looks like it will be later. There is still much documentation work to be done.

**Reply by omprakashchavan01** [reply]
Got it. Thank you.

**Comment by mochitest** [comment]
When will Appium 2.0 officially release?

**Reply by jlipps** [reply]
When it is ready.

**Comment by v-mwalk** [comment]
Hi guys; any thoughts on 2.0 release date?

I'm using the beta-25 (and all is good - so far :-) ) but my boss is asking why I have to use it and when it will come out of beta.  I have to use it for the limitXPathContextScope setting (and because loving the decoupling lol) but would love to give him an answer..

**Reply by jlipps** [reply]
the main thing we're waiting on for a final release is a new website and docs structure. as far as code goes it's basically done. but it doesn't feel right to make an official release until the doc situation is in a good place. maybe that will help your boss feel better about things.

**Comment by dgilbert-intellum** [comment]
So I will also ask, as I am struggling a bit, are we officially released yet?  And if so, is the doc considered complete, because I am struggling with so much incorrect information from the web that my ability to get any real work accomplished has tanked.  Is there an active online community of other QA professionals using 2.0 yet that I could join?  Would love to find that... really struggling to wade through the billions of "How do I do phone test use appium help pls" questions looking for real help from anyone with real experience.  Thanks.

**Reply by RSM-SBA** [reply]
i would say that the appium forum is a good place to find professionals using 2.0
https://discuss.appium.io/

**Reply by jlipps** [reply]
Appium 2.0 documentation is very much NOT complete, that is the main reason 2.0 is not yet officially released. the in-progress docs are here: https://appium.github.io/appium/docs/en/2.0/. the most relevant part for people who already know Appium 1.x is the [migration guide](https://appium.github.io/appium/docs/en/2.0/guides/migrating-1-to-2/).

**Comment by 4nif** [comment]
Do we have ETA for the appium 2 release date?

**Reply by jlipps** [reply]
Appium 2 is already ready and you should already be using it. We're no longer supporting Appium 1. We're just working on docs now for Appium 2. Can't have an official release without decent docs.

**Comment by jbtech123** [comment]
hello, is there any update on official release date?

**Reply by jlipps** [reply]
as soon as the documentation is done. from a code perspective you should already be using appium 2.

**Reply by jbtech123** [reply]
@jlipps thanks for reply, is appium2 image is available on dockerhub?

**Reply by jlipps** [reply]
the core appium team doesn't support or maintain anything docker related, so i have no idea.

**Comment by vesper8** [comment]
Any ETA at all when the docs / new website will be ready?

And what will be the URL of these? Is there a "work in progress" documentation that's being worked on / updated that we can look at now?

Thanks! It feels like it's getting near!

**Reply by KazuCocoa** [reply]
https://appium.github.io/appium/docs/en/2.0/
Links in https://github.com/appium/appium lead the page.

**Reply by jlipps** [reply]
Prose docs are basically done. Also almost done with auto generated command documentation. There's really no reason not to be using Appium 2 or referencing these docs at this point!

**Comment by shreyansh791** [comment]


**Reply by jlipps** [reply]
@shreyansh1991 I refer you to the various answers already given on this page

**Comment by shreyansh791** [comment]
hello, is there any update on official release date?

**Reply by jlipps** [reply]
as soon as the documentation generation is done. there's no reason not to use it now, though
